ntoskrnl/se/token.c

```diff
@@ -1101,6 +1101,264 @@ SepFindPrimaryGroupAndDefaultOwner(
     return STATUS_SUCCESS;
 }
 
+/**
+ * @brief
+ * Internal private function that returns an opened handle
+ * of an access token associated with a thread.
+ *
+ * @param[in] Thread
+ * A pointer to a Executive thread (ETHREAD). This parameter
```

Comment:
```suggestion
 * A pointer to a Executive thread. This parameter
```
Kernel devs know what it is (and its type, it's just written below in the function parameters list).